code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1502,6 +1503,75 @@
 	reagents.add_reagent(NUTRIMENT, 8)
 	bitesize = 2
 
+/obj/item/weapon/reagent_containers/food/snacks/skitter/ //if ye dish is a child of skitter it will move around after 30 ticks
+	name = "skittering burger"							//make sure to add "processing_objects += src" under New() for it to work
```

Comment:
everything looks good, although you should be able to add it to processing_objects at this level instead of just on the subtypes; all subtypes must call ..()

Reply:
Would 
```
/obj/item/weapon/reagent_containers/food/snacks/skitter/New()
	processing_objects += src
```
do?
Or do I need to call extra stuff on the subtypes?

Reply:
that's fine, but remember in new you must at some point call ..() or you will fail linters

Reply:
I'll just copypaste what you posted there.